### PR TITLE
#16440 Include description column when exporting as XLSX

### LINKED ISSUE #16440: Export column description in XLSX
**Is your feature request related to a problem? Please describe.**

I want export data with database column comment replace dafault header.Such as export excel header,show database column comment not the colum name.

**Describe the solution you'd like**

Export data show database column comment.

### PR DESCRIPTION
**Pull request by ShadelessFox**
image

### CODE REVIEW

File: plugins/org.jkiss.dbeaver.data.office/OSGI-INF/l10n/bundle_pt_BR.properties [hunk 1]
```diff
@@ -17,7 +15,7 @@ dataTransfer.processor.xlsx.property.exportSql.description=Exportar SQL para uma
 dataTransfer.processor.xlsx.property.extension.label = Extens\u00f5es de arquivos
 dataTransfer.processor.xlsx.property.splitSqlText.name=Dividir o texto SQL
 dataTransfer.processor.xlsx.property.splitSqlText.description=Dividir SQL exportado em linhas por CR
-dataTransfer.processor.xlsx.property.headerFont.name=Fonte da linha do cabe�alho
+dataTransfer.processor.xlsx.property.headerFont.name=Fonte da linha do cabe�alho
```
Comment (LonwoLonwo) [inline plugins/org.jkiss.dbeaver.data.office/OSGI-INF/l10n/bundle_pt_BR.properties]: cabeï¿½alho - it looks strange. Is it expected?

File: plugins/org.jkiss.dbeaver.data.office/OSGI-INF/l10n/bundle_pt_BR.properties [hunk 2]
```diff
@@ -28,4 +26,4 @@ dataTransfer.processor.xlsx.property.splitByRowCount.description=Dividir por con
 dataTransfer.processor.xlsx.property.splitByColNum.name=Coluna do grupo
 dataTransfer.processor.xlsx.property.splitByColNum.description=N\u00famero da coluna para agrupar linhas na folha por valor da coluna
 dataTransfer.processor.xlsx.property.dateFormat.name=Formato de dados excel
-dataTransfer.processor.xlsx.property.dateFormat.description=Formato de excel e datas (e.x. m/d/yy hh:mm) vo�e pode mudar no excel.
\ No newline at end of file
+dataTransfer.processor.xlsx.property.dateFormat.description=Formato de excel e datas (e.x. m/d/yy hh:mm) vo�e pode mudar no excel.
```
Comment (E1izabeth) [inline plugins/org.jkiss.dbeaver.data.office/OSGI-INF/l10n/bundle_pt_BR.properties]: This and mentioned by LonwoLonwo line seems accidentally modified. Please, return it back.

**Review by E1izabeth — CHANGES_REQUESTED** [review]

**Review by E1izabeth — APPROVED** [review]

**Review by LonwoLonwo — APPROVED** [review]

**Comment by uslss** [comment]
in case not all of the columns had description, but only description is chosen in export settings, columns without descriptions are incorrectly shown in excel
image

at the same time if none header format is chosen column width is set more correctly
image

**Comment by DariaMarkaryan** [comment]
no header is added when we export with description as header and at least one of these settings checked 
image

**Pull request merged**